docs/errorhandling.rst

```diff
@@ -28,6 +28,59 @@ exception to the :attr:`~flask.Flask.logger`.
 But there is more you can do, and we will cover some better setups to deal
 with errors.
 
+.. _error-handlers:
+
+Error handlers
+--------------
+
+.. versionadded:: 1.0
+
+Some errors are unavoidable because not only you make them.  Users might
```

Comment:
i thought that coming here directly from the quickstart would be good.

also this is the introduction for the “i want to show an error page” use case. the other one generally for errors.

which parts do you think should be moved or removed?